src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -4568,6 +4610,19 @@ SpinelNCPInstance::handle_ncp_spinel_value_is(spinel_prop_key_t key, const uint8
 		if (len > 0) {
 			syslog(LOG_NOTICE, "[-NCP-]: RCP is running \"%s\"", rcp_version);
 		}
+	} else if (key == SPINEL_PROP_THREAD_NETWORK_TIME) {
+		boost::any value;
+		NetworkTimeUpdate update;
+
+		if (unpack_thread_network_time(value_data_ptr, value_data_len, value, true) == kWPANTUNDStatus_Ok && 
+			extract_thread_network_time(value, update)) {
+			
+			syslog(LOG_INFO, "[-NCP-]: Network time update: network_time:%lu, status:%d", update.mNetworkTime, update.mStatus);
+
+			handle_network_time_update(update);
```

Comment:
I think if we change the `handle_network_time_update()` to use `const ValueMap &` as input parameter, it would make this simpler as we won't need the `extract_thread_network_time()` here and can directly pass the value map.

For logging we can 
- either use the valuemap and check the entries under the keys `kWPANTUNDValueMapKey_TimeSync_Time` and `kWPANTUNDValueMapKey_TimeSync_Status`, 
- or call `unpack_thread_network_time()` with `as_val_map` parameter as `false` and log the resulting string.